src/python/grpcio_tests/tests/_runner.py

```diff
@@ -181,6 +187,13 @@ def try_set_handler(name, handler):
         # Run the tests
         result.startTestRun()
         for augmented_case in augmented_cases:
+            skip_test = False
```

Comment:
Move this assignment to an `else:` attached to the `for:`.

(Then see if the local field doesn't just evaporate entirely if the `continue` is also moved into the `else:`?)

Reply:
Changed